### ISSUE TITLE
Live completely moneyless

### ISSUE THREAD

**Issue by balupton** [balupton/meta#4]
Essentials:
- [ ] Call up every single organic grocer in sydney to see if we can pickup their leftovers
  - [ ] Requires phone plan, which I do not currently have

Possibilities:
- [ ] SOMEHOW find free rent with 2 or 3 bedrooms available (me+helen, hayden, imogen)
- [ ] SOMEHOW figure out how to get free utilities on the premise
  - [ ] internet
  - [ ] electricity
  - [ ] water

Difficulties?
- [x] Liquidate Bevry to avoid paying tax, banking, and company registration fees. Receive any money via BitCoin
  - This could really screw us over in terms of receiving donations etc... and IP issues
- [x] Cancel phone plans
- [x] Get rid of car to avoid insurance fees and to be more environmentally friendly
  - Hitchhike and public transit instead... at least until we stumble upon a mission motorcycle or tesla
- [x] Get rid of health insurance?

Questions
- What to do if we receive more food than we can dispose of? Having a system that tracks everyones disposables, and issues an alert to everyone on a first come first serve reserve system when they are disposing, would be awesome

**Comment by balupton** [comment]
@ahdinosaur @elf-pavlik any suggestions? know anyone in Sydney who can help with this? know anyone I can connect to?

Right now, I'm running entirely off http://www.moneylessmanifesto.org

**Comment by ahdinosaur** [comment]
@balupton, the list you have seems good to me. i've only been able to live without paying for shelter either by living out of a car in front of a friendly home or by couch-surfing at friendly homes, but only as one person. a promising option is to live on rural eco-villages, as costs can become sustainable.

i don't know anyone in Syndney and the only moneyless person i know is @elf-pavlik, so let's hope elf can help teach us how to live completely moneyless. :)

**Comment by balupton** [comment]
> a promising option is to live on rural eco-villages, as costs can become sustainable.

Interesting. I'll look into that.

One thing that needs to be done is outline the rules and ethics of the game. For instance:
- Is receiving money okay?
- Is directing other people's money to useful services okay?
- What about having others pay for our rent? Is relinquishing that power to them good? Or is that missing the point of moneyless living?

**Comment by elf-pavlik** [comment]
Hey @balupton, I'd <3 to work together with you on digital tools which help with such transition!!!
1. **Transparency** - until we can put in place solid alternatives to current finances, which **do scale**, we can't really participate in use of modern technologies without someone else still using money in a process :unamused: I see it as transitional flow and keeping public, transparent logs of all such indirect dependencies on money can help us remove them one by one + enable _Collective Inteligence_ + lay down foundations for [Holopticism](https://www.youtube.com/watch?v=GY7KHH1EbEc) BTW glad to see fellow _holon_ here :smile: @ahdinosaur 
2. **Money Deprecation** - i already signaled on [Web Payments](http://web-payments.org) mailing list need for making possible for people to add _deprecation warnings_ on any currency, while transferring it. _see [mail in W3C lists archive](http://lists.w3.org/Archives/Public/public-webpayments/2014Jan/0012.html)_ ASAP I will start requiring issuing and publishing such warning whenever someone uses money to secure my access to any real world asset (product/service)
3. **Web Currencies** - i also made place holder to work on [Web Currencies](http://www.w3.org/community/webcurrencies/) which will build on top of _Web Payments_. Imagine that someone contributing to harvesting renewable energy could transfer you some **kWh of electricity** over something like _moneyless gittip_, also **days of coworking** in networks like http://copass.org (I know people who founded CoPass!) and so on. Those variants relate to what I refer to as _shared benefit_ (may rename to _assets shares_) in [Polyeconomy](http://polyeconomy.info)
4. When it comes to food, friends have develop very cool platform to coordinate reducing food waste and receiving food surplus directly from shops. For now they only have it in german language but they work on support for multiple languages as well as open sourcing it! http://www.lebensmittelretten.de (online translation your friend for now...). I also know someone from http://www.zerowaste.sa.gov.au/ who most likely would support porting it there!

I guess enough for now... let's keep this ball rolling :8ball:

**Comment by elf-pavlik** [comment]
something else you may like to keep an eye on - http://unmonastery.org
i participated in pilot session of the first one in Matera, working on #ZeroWaste, connecting with #CodeForAll and some other social projects - https://vimeo.com/album/2804146/video/90471443

**Comment by dominictarr** [comment]
Hey this is really interesting to see. Now while I havn't lived without money, I have ventured down the road _towards_ this. In various periods in my life i drastically minimized my need for money, this was in part pragmatic, and part political statement.

I think it's absurd that, for example, people spend decades paying off a morgage, or that people working some shitty job (and I've had those too) might pay 80% of their income back out as rent or bills. Even if you where able to save 50% of your income while maintaining a modern lifestyle that would be pretty good.

My goal was to live on 20% of my income. This meant for every week worked I could spend an entire month doing pretty much whatever I wanted. I achieved this mainly by eliminating rent. Initially, I did hospitality work (washing dishes), later I got a part time low level tech job (administering a really terrible cms).

you could say that living _without_ money is like being vegan, but living with very little money is like being vegeterian

I lived in the forest in wellington, NZ. I would shower and wash my clothes at friends places.
I have also lived in boats in multiple periods, and I spent several months living in a car in Australia. Australia has really good 3g coverage. It also lots of sun. I've been day dreaming about doing a crusing version of the world solar challenge: http://www.worldsolarchallenge.org/

It wouldn't be a race - if solar could get me 200km a day that is double what you'd do by bicycle so that would be great. To get the vehicle road-legal the simplest way would probably be to make sure it was still legally classifiable as a bicycle.

I think it's very important to be upfront about what you are doing. People will be very interested in what you are doing and your reasons for doing it. Because of this people
where always quite happy for me to use their shower. Here is another example:
http://www.gizchina.com/2010/12/01/beijing-design-graduate-lives-2-meter-egg/
Notice how the novelty of his design completely changes the meaning of his lifestyle.
He's essentially living in a shack on a street, but since his shack is a design piece it
does not fit into the predefined social classes. He is living like a poor person but acting
like a rich person. In fact, he has become much better off by rejecting rent, and has also become an _Interesting Person_ by doing so in a novel way.

In other words, presentation is important.

**Comment by balupton** [comment]
Sweet. Thanks for the awesome feedback!

Those are some interesting insights on the cross-barrier between moneyless and tech.

I've been toying with an alternation of the [Startup Hostel](http://startuphostel.org) idea, where we fly in a bunch of prominent startup people (@dominictarr you were one of those I had imagined) into a villa in a 3rd world country, all expenses paid, then sell accommodation and training tickets to students who want to learn from the prominent residents. The requirement of this is to evaluate whether or not I can convince a bunch of prominent people to live together for free, and then convince a bunch of students to pay for to be near or to learn from such people. Dominic, what are your thoughts on such an arrangement?

I really liked the comparison to veganism and vegetarianism, I hadn't thought of that before despite suspecting something along those lines due to the unstoppable cognitive dissonance that I feel using money, hence the urgency to, in @elf-pavlik's words "leave this crazy system"

Another concept I've been toying with is sponsored living, in which a few companies pay say $150/month for the return of sponsorship in terms of adverts in project readmes, talks, websites, etc.

And a final concept I've been toying with, is just having a wishlist where someone can buy or donate the money items you need, in exchange for a forever permanent thankyou on a website or whatever, or in return for something else.

I should have access to a phone tomorrow, so I'm keen to start calling up grocers and getting a lay of the land here. I'm also keen to setup a freegan community here in Sydney, however the meetup.com $100/year price tag is a bit of a deterrent when you're living moneyless.

**Comment by elf-pavlik** [comment]
+1000 for wishlists!!!

I started toying with my homepage to  soon use [5 star linked data](http://5stardata.info/) for publishing my wishes. I find it crucial that we don't depend on central silo for that and everyone can easily syndicate / aggregate wishes anywhere.

Not sure if you have experience with RDF, you could for example copy the whole source of [my homepage](https://wwelves.org/perpetual-tripper) and paste it into http://rdfa.info/play to see embedded graph. To see JSON-LD version just negotiate it:

``` shell
curl -H "Accept: application/ld+json" https://wwelves.org/perpetual-tripper/
```

RDF has various peculiarities which requires tricks like [hydra:Collection](https://www.w3.org/community/hydra/wiki/Collection_Design) to publish long wishlists in sane way. I plan to make an [Unhosted](http://unhosted) wishlist generator using something like http://jeremydorn.com/json-editor/ ... some random thoughts for similar personal profile generator you can find in [this slidedeck](https://docs.google.com/presentation/d/1EECDxH2Xe2rAG7XR64OVdt1PTSG-cF3DIgvFQbJv1cs/edit#slide=id.p) and [prove of concept app](https://github.com/elf-pavlik/plp-generator) (with _[remotestorage](http://remotestorage.io) backend in one of pull requests_)

I also hope to start coordinating my work with projects like
- http://www.tradeschool.coop/ | https://github.com/orzubalsky/tradeschool
- https://embassynetwork.com/ | https://github.com/jessykate/modernomad

etc.

If you like we could collaborate remotely on a way to publish and discover wishes with some syndication/aggregation patterns!

**Comment by balupton** [comment]
@elf-pavlik cool, this is my current idea for it https://github.com/ideashare/ideas/issues/12 — not sure if that aligns with your target, perhaps? curious to hear your feedback on the app, as it's consumed my attention the past weeks. I guess my take is on things in excess, and your take is on things in scarcity right?

**Comment by balupton** [comment]
I've now accomplished free food. I'm now volunteering twice a week at a community run vegan cafe, and I bring enough containers for left-overs to last the week :-) Woot woot.

Now just rent and utilities to figure out.

**Comment by dominictarr** [comment]
could you get away with living in a boat in sydney harbor?
of course, you'd need to buy a boat is it okay to spend money to avoid money?

**Comment by dominictarr** [comment]
sorry, boat is probably a bad idea, because you'll need to maintain it, and that will take time & money.

**Comment by ahdinosaur** [comment]
congrats @balupton on the food, that's great! :)

**Comment by balupton** [comment]
Closing. Gave a talk about my experience at 
https://www.youtube.com/watch?v=Lt_oKuaFgrg&list=PLYVl5EnzwqsRtF328lLu_sjJC7FxO4B1S&index=3
